egs/wsj/s5/steps/nnet3/tdnn/train.sh

```diff
@@ -5,7 +5,7 @@
 # Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
 #           2013  Xiaohui Zhang
 #           2013  Guoguo Chen
-#           2014  Vimal Manohar
+#           2014-2016  Vimal Manohar
```

Comment:
that would be OK with me, but try to simplify it by cutting out unnecessary
code.

On Thu, Mar 3, 2016 at 12:25 AM, Vimal Manohar <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/nnet3/tdnn/train.sh
> <https://github.com/kaldi-asr/kaldi/pull/539#discussion_r54837404>:
>
> > @@ -5,7 +5,7 @@
> >  # Copyright 2012-2015  Johns Hopkins University (Author: Daniel Povey).
> >  #           2013  Xiaohui Zhang
> >  #           2013  Guoguo Chen
> > -#           2014  Vimal Manohar
> > +#           2014-2016  Vimal Manohar
>
> I can add the changes to a new script called nnet3/tdnn/train_raw.sh.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/539/files#r54837404>.
>